Templates/DefaultProject/Template/.gitignore

```diff
@@ -3,3 +3,4 @@
 [Uu]ser/
 [Uu]ser_test*/
 _savebackup/
+CMakeUserPresets.json
```

Comment:
I wonder, maybe we should include this in the main engine `.gitignore` also for anyone who might provide a local `CMakeUserPresets.json` for the engine?

Reply:
Ah looks like it's already there.